packages/cli/src/Server.ts

```diff
@@ -1881,121 +1884,119 @@ class App {
 		// Authorize OAuth Data
 		this.app.get(
 			`/${this.restEndpoint}/oauth1-credential/auth`,
-			ResponseHelper.send(async (req: express.Request, res: express.Response): Promise<string> => {
-				if (req.query.id === undefined) {
-					res.status(500).send('Required credential id is missing!');
-					return '';
-				}
+			ResponseHelper.send(
+				async (req: OAuthRequest.OAuth1CredentialAuth, res: express.Response): Promise<string> => {
+					const { id: credentialId } = req.query;
 
-				const queryBuilder = Db.collections.Credentials!.createQueryBuilder('c');
-				queryBuilder.andWhere('c.id = :id', { id: req.query.id });
-				queryBuilder.innerJoin('c.shared', 'shared');
-				queryBuilder.andWhere('shared.userId = :userId', {
-					userId: (req.user as User).id,
-				});
+					if (!credentialId) {
+						res.status(500).send('Required credential ID is missing!');
```

Comment:
We are using `ResponseHelper.send` in the function signature, so you can simply use `throw new ResponseHelper.ResponseError('Required credential ID is missing!', undefined, 500);` and it should work.